tests/infra/node.py

```diff
@@ -266,7 +266,7 @@ def client(self, identity=None, **kwargs):
             if identity
             else None,
             "ca": os.path.join(self.common_dir, "networkcert.pem"),
-            "description": f"node {self.node_id} as {identity or 'unauthenticated'}",
+            "description": f"[{self.node_id}|{identity or '?'}]",
```

Comment:
`?` looks a little confusing for unauthenticated connections in the logs `2020-09-15 16:32:35.589 | INFO     | infra.node:wait_for_node_to_join:247 - [2|?] GET /node/commit`. Perhaps stick to "unauthenticated" or "anonymous"?

Reply:
That's super long, and extremely common. I do not want to print lots of "unauthenticated" that force me to truncate interesting payloads.